test/unit/rules/no-scope-outside-table-headings-test.js

```diff
@@ -0,0 +1,53 @@
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
+  name: 'no-scope-outside-table-headings',
+
+  config: true,
+
+  good: [
+    '<th scope="row">Some table heading></th>',
+    `
+<table>
+<th scope="col">Table header</th>
+<td>Some data</td>
+</table>
+`,
+    '<CustomComponent scope />',
+    '<CustomComponent scope={{foo}} />',
+  ],
+  bad: [
```

Comment:
Can we add this to bad just to show that we don't care what the value is?
* `<td scope>`